app/managers.py

```diff
@@ -0,0 +1,40 @@
+import sqlite3
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("../cinema.sqlite")

```

Comment:
Consider using a context manager (with statement) for the database connection to ensure that the connection is properly closed after operations are completed.